### PR TITLE
Added my profile in data and default user image URL

### PR DESCRIPTION
**Pull request by hemantwasthere**
## Changes proposed

1.  I have added my profile data in `data` directory in `hemantwasthere.json` file
2.  I have added a default user image if in any case user doesn't provide an avatar URL
3.  For that default user image I had to add the URL domain name in `next.confing.js`

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1617"></a>

### CODE REVIEW

File: components/user/UserPreview.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default function UserPreview({ profile }) {
       className="flex gap-x-6 rounded-full border-2 border-gray-200 hover:border-gray-500 p-4 my-2"
     >
       <Image
-        src={profile.avatar}
+        src={profile.avatar ? profile.avatar : "https://d1aettbyeyfilo.cloudfront.net/eddiejaoude/19083469_1619470266Fpchubber-hi.webp"}
```
Comment (Panquesito7) [inline components/user/UserPreview.js:10]: Please revert the changes in this file. Let us know if you need any help with that! 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

File: next.config.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ const nextConfig = {
       "avatars.githubusercontent.com",
       "user-images.githubusercontent.com",
       "cdn.nhcarrigan.com",
+      "d1aettbyeyfilo.cloudfront.net"
```
Comment (Panquesito7) [inline next.config.js:10]: Please revert the changes in this file. Let us know if you need any help!


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: components/user/UserPreview.js (hunk 1 above)
Comment (hemantwasthere) [reply components/user/UserPreview.js]: Done that 😀

Comment (hemantwasthere) [reply components/user/UserPreview.js]: @Panquesito7 hey is my code good to be get merged

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please move your file to `public/data/`. Let us know if you need any help with that.


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by hemantwasthere** [comment]
> Please move your file to `public/data/`. Let us know if you need any help with that.
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix


This is nextjs branch there is no public directory
